src/types/form.ts

```diff
@@ -8,8 +8,3 @@ export interface DefaultFieldProps {
   text?: string;
   children?: any;
 }
-
```

Comment:
Can you explain what happened to FormMode? How do you know if it is winter or berth currently, for example in BerthPage?

Reply:
Pages are no longer shared for both types of applications. It starts from the topmost container, it renders either a berth or a winter-storage.
We only share "micro" components if possible.